protocol/communities/manager.go

```diff
@@ -1924,6 +1924,10 @@ func (m *Manager) JoinCommunity(id types.HexBytes) (*Community, error) {
 	if community == nil {
 		return nil, ErrOrgNotFound
 	}
+	if community.Joined() {
+		// Nothing to do, we are already joined
+		return community, ErrOrgAlreadyJoined
```

Comment:
Based on error processing in this method, we not expect that community will be returned with error?

Reply:
Also, test not expect it, so maybe good to send `nil`?